poleno/utils/tests/test_forms.py

```diff
@@ -272,7 +272,6 @@ def _render(self, template, **context):
     def setUp(self):
         self.tempdir = TempDirectory()
         self.settings_override = override_settings(
-            TEMPLATE_LOADERS=(u'django.template.loaders.filesystem.Loader',),
```

Comment:
Tym ze sme prepisali `TEMPLATE_LOADERS`, django nebol schopny najst template `utils/forms/compositetextfield/finalize.txt`. `u'django.template.loaders.filesystem.Loader'` je zahrnuty defaultne.

Reply:
@viliambalaz asi lepsie by bolo tu v `override_settings` zahrnut len prave tie loadre, ktore potrebujeme. Cize pridaj tu loader, ktory chyba.

Reply:
:ok:

Reply:
@viliambalaz Ten `poleno.utils.template.TranslationLoader` tam potrebujes? V tychto testoch nikde nerobime s lokalizovanymi templatami.

Reply:
:ok:

Reply:
@viliambalaz `TEMPLATE_LOADERS` mozes zapisat iba ako tuple stringov, nemusis to pisat ako tuple tuplov stringov. Tj.:

```
TEMPLATE_LOADERS=(u'django.template.loaders.filesystem.Loader', u'django.template.loaders.app_directories.Loader')
```

Rovnako na ostatnych miestach v PR.

Reply:
:ok: Zamenil som si to s `TranslationLoader`, kde je vnoreny tuple.